docs/installation.adoc

```diff
@@ -154,3 +155,10 @@ __  ____  __  _____   ___  __ ____  ______
 2021-03-05 12:26:43,200 INFO  com.eno.kub.ope.Operator - Adding listener 'OperatorXpClientCacheInvalidate'
 2021-03-05 12:26:43,200 INFO  com.eno.kub.ope.Operator - Starting informers
 ----
+
+=== Uninstall xp from your cluster
```

Comment:
This title is still confusing. We have to differentiate between the xp-operator and xp. 

You do not need to uninstall the operator to redeploy some XP7 deployment. Simplest way is to delete the namespace with the deployment (or if other stuff is in that namespace, then delete the deployment + pvc etc) and create again.

Uninstalling the xp-operator is another, somewhat unrelated operation.

So I guess my question is: What are you describing in this section?

Reply:
I am referring to both. If someone wants to delete xp7deployment and the operator.